hutch_python/constants.py

```diff
@@ -4,7 +4,7 @@
 
 CONDA_BASE = Path('/reg/g/pcds/pyps/conda/py36')
 
-CUR_EXP_SCRIPT = '/reg/g/pcds/engineering_tools/{0}/scripts/get_curr_exp {0}'
+CUR_EXP_SCRIPT = '/reg/g/pcds/engineering_tools/{0}/scripts/get_curr_exp -H {0}'
```

Comment:
Linter says the line is too long
```suggestion
ENG_TOOLS = '/reg/g/pcds/engineering_tools/'
CUR_EXP_SCRIPT = ENG_TOOLS + '{0}/scripts/get_curr_exp -H {0}'
```